test/reline/test_key_actor_emacs.rb

```diff
@@ -1436,4 +1436,9 @@ def test_unix_line_discard
     input_keys("\C-f\C-u", false)
     assert_line_around_cursor('', '')
   end
+
+  def test_vi_editing_mode
+    @line_editor.__send__(:vi_editing_mode, nil)
```

Comment:
When key binding is set, `key.char` is passed as `symbol` to `LineEditor#update`.
I can't seem to test it properly with `input_keys`. If you have a better way to do this, please let me know.